Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -73,22 +88,54 @@ struct LoginView: View {
     }
 
     private func componentSetup() -> some View {
-        VStack(alignment: .center, spacing: 0.0) {
-            TextField("", text: $email)
-                .modifier(PlaceholderModifier(showPlaceHolder: email.isEmpty, placeholder: "Email"))
-                .padding(.bottom, 20.0)
-
-            SecureField("", text: $password)
-                .modifier(PlaceholderModifier(showPlaceHolder: password.isEmpty, placeholder: "Password"))
-                .padding(.bottom, 20.0)
-
-            SButtonView(loginAction: {
-                logInTrigger.send(())
-            }, title: loginTitle)
+        VStack(alignment: .leading, spacing: 0.0) {
+            emailSetup()
+            passwordSetup()
+            SButtonView(
+                isValid: $output.isLoginEnabled,
+                loginAction: { logInTrigger.send(()) },
+                title: loginTitle
+            )
         }
         .padding([.leading, .trailing], 24.0)
     }
 
+    private func emailSetup() -> some View {
+        VStack(alignment: .leading, spacing: 0) {
```

Comment:
Let's make this decimal
```suggestion
        VStack(alignment: .leading, spacing: 0.0) {
```